example/src/App.js

```diff
@@ -9,10 +10,15 @@ function App() {
         <Route path="/cm">
           <CM />
         </Route>
+
+        <Route path="/cm/edit">
+          <EditViewPage />
+        </Route>
+
+        <Route path="/settings">
```

Comment:
I think the path should be `/settings/:settingType` so you can redirect the user to the application setting when the user clicks on the Setting link from the main menu by using en effect in the SettingsPage or another component.